source/features/conversation-activity-filter.tsx

```diff
@@ -125,10 +125,10 @@ async function addWidget(header: string, state: State): Promise<void> {
 	position.classList.add('rgh-conversation-activity-filter');
 	position.after(
 		<details
-			className={`details-reset details-overlay d-inline-block position-relative ${dropdownClass}`}
+			className={`details-reset details-overlay d-inline-block ml-2 position-relative ${dropdownClass}`}
```

Comment:
Fixed the spacing of the elements added by RGH.

Before:

![before](https://user-images.githubusercontent.com/46634000/161035074-7d1e2e31-8e4c-4177-bf3e-733a54a0c2e1.png)

After:

![after](https://user-images.githubusercontent.com/46634000/161035048-54053268-924c-4664-ae91-32162481b72f.png)